docs/developer/go-style-guide.md

```diff
@@ -413,27 +413,27 @@ FAVOR:
 
 ```go
 type Object struct{
-subobject SubObject
+    subobject SubObject
```

Comment:
is it too much spacing here? and in line  430?

Reply:
yep, fixed